helix-term/src/health.rs

```diff
@@ -314,6 +316,20 @@ pub fn language(lang_str: String) -> std::io::Result<()> {
     Ok(())
 }
 
+fn probe_parser(grammar_name: &str) -> std::io::Result<()> {
+    let stdout = std::io::stdout();
+    let mut stdout = stdout.lock();
+
+    use helix_loader::grammar::get_language;
+
+    write!(stdout, "Tree-sitter parser: ")?;
+
+    match get_language(grammar_name) {
```

Comment:
Instead of a `use` that is used once you can call the function with it's fully qualified path: `match helix_loader::grammar::get_language(grammar_name) { .. }`